src/redux/services/resourceRefs.ts

```diff
@@ -215,6 +215,27 @@ function cleanResourceRefs(resources: K8sResource[]) {
   });
 }
 
+/**
+ * Removes all refs to the specified resource from other resources
+ */
+
+export function removeReferringRefs(resource: K8sResource, resourceMap: ResourceMapType) {
```

Comment:
shouldn't we transform the refs to unsatisfied instead of removing them? - and of course keep in mind the "optional" sibling

Reply:
yes - nice catch! will fix...